libs/base/ui/overlays/notification-center/src/registry.controller.ts

```diff
@@ -1,142 +1,110 @@
 import { LitElement, ReactiveController } from 'lit';
-import { Schemes } from '../../notification/src';
-import {
-  NotificationRegistry,
-  NotificationStrategy,
-} from './notification-center.model';
-
-const defaultStrategy: NotificationStrategy = {
-  scheme: Schemes.LIGHT,
-  autoClose: true,
-  autoCloseTime: 8000,
-  closable: true,
-  floating: true,
-};
-
-const createKey = (registry: NotificationRegistry[]): string => {
-  return String(
-    Number(
-      registry
-        .filter(({ key }) => !!key)
-        .sort((a, b) => Number(b.key) - Number(a.key))[0]?.key || Date.now()
-    ) + 1
-  );
-};
+import { NotificationRegistry } from './notification-center.model';
 
-export class RegistryController implements ReactiveController {
-  private _registry: NotificationRegistry[] = [];
-  private _delayedCallbacks: { [key: string]: ReturnType<typeof setTimeout> } =
-    {};
-
-  set registry(newRegistry: NotificationRegistry[]) {
-    //need to add keys and delayed callback to the new Notification items
-    const newRegistryWithParams: NotificationRegistry[] = newRegistry.map(
-      (item): NotificationRegistry => {
-        if (!item.key) {
-          //get last key or generate new one
-          const key = createKey(this.registry);
-
-          const mergedStrategy = {
-            ...defaultStrategy,
-            ...item,
-          };
-
-          //add auto-close handler
-          if (mergedStrategy.autoClose) {
-            this._delayedCallbacks[key] = setTimeout(() => {
-              this.toggleVisibility(key);
-            }, mergedStrategy.autoCloseTime);
-          }
-          return {
-            ...mergedStrategy,
-            key,
-            visible: false,
-          };
-        }
-
-        return item;
-      }
-    );
+import { AlertType } from '@spryker-oryx/ui';
+import { Notification } from '../../notification';
 
-    this._registry = newRegistryWithParams;
-    this.host.requestUpdate();
-  }
+const DESTROY_DELAY_TIME = 600;
```

Comment:
maybe this should be a property with default value?

Reply:
What about changing it to 1s to allow for enough time. I'd rather keep the implementation simple here, more time doesn't harm.